frontend/src/components/MyNavbar/MyNavbar.js

```diff
@@ -0,0 +1,84 @@
+import React, {useState} from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './MyNavbar.css'
+import {
+    Navbar,
+    Button,
+    ButtonDropdown,
+    DropdownItem,
+    DropdownMenu,
+    DropdownToggle
+  } from 'reactstrap';
+
+  import ButtonSpacer from '../ButtonSpacer/ButtonSpacer'
+
+
+const LinkButton = (props) => {
+    return(
+        <>
+        <Button color="info" id="button-background" href = {props.url}>
+           <text className="button-text">{props.label}</text>
+        </Button>
+        <ButtonSpacer/>
+        </>
+    )
+}
```

Comment:
Now that we are removing the ButtonSpacer component, what do you think about putting the button inside of the Navbar instead.

Reply:
I am not sure what you mean by this.  Do you mean that I should remove the LinkButton component and directly use a button in the Navbar component?

Reply:
Yes, since the only things that will be used in the LinkButton is the Button, what do you think about placing it in the navbar directly.

Reply:
I made LinkButton to easily change all buttons at once if needed, but I can remove it.

Reply:
Ok great idea, lets keep it. For simplicity sake, can we have one component per file. Now that we are keeping this, can we move it to a separate file and import it into this file.

Reply:
Done